tsconfig.json

```diff
@@ -9,8 +9,8 @@
     "compilerOptions": {
         "allowSyntheticDefaultImports": true,
         "incremental": true,
-        "lib": ["dom"],
-        "module": "commonjs",
+        "lib": ["dom", "ESNext"],
```

Comment:
Lets push this change directly to master or to 13.1.0. Lets not have it part of this PR